### PR TITLE
#169861112 update readme.md with keyvault keys

### PR DESCRIPTION
**Pull request by rchessa**

### CODE REVIEW

**Comment by ioggstream** [comment]
@lucaprete do we have some guideline for variables? No snake case? mylongvariablenamewithoutseparators seems error prone

**Comment by LucaPrete** [comment]
I would use - to separate words

**Comment by ioggstream** [comment]
Let's do it then ;)

File: README.md [hunk 1]
```diff
@@ -154,10 +154,44 @@ cd ../dns_zone_private_common && terragrunt apply
...
 cd ../key_vault && terragrunt apply
 
-# MANUAL OPERATIONS REQUIRED: the following secrets need to be manually
-# inserted in the Azure Keyvault 
-# 
-# * TBD - look at the actual Keyvault for more info
+# MANUAL OPERATIONS REQUIRED: the following secrets need to be manually inserted in the Azure Keyvault 
+Secret  | Description 
+------- | --------------
+app-insight-web-tests-Ocp-Apim-Subscription-Key | to be defined
```
Comment (LucaPrete) [inline README.md]: The APIM secret used by the monitoring user (for automated monitoring activities). This is usually the user "IO Monitoring" user, listed under users in the APIM

Comment (rchessa) [reply README.md]: Added

**Pull request merged**